ibis/backends/pandas/tests/execution/conftest.py

```diff
@@ -144,7 +144,7 @@ def time_df3():
                     start='2017-01-02 01:02:03.234', periods=8
                 ).values
             ),
-            'id': list(range(1, 9)),
+            'id': list(range(1, 5)) * 2,
```

Comment:
Why changing this?

Reply:
This is meant to extend the test cases to show that `group_by` parameter in window could actually changed the row ordering.  When `id` are all distinct, `groupby` produces a result Dataframe where the row ordering is the same with the case where there is no `groupby`, but with this change, when we group by `id` in tests, the udf result is not guranteed to be ordered by `time`.  A new test is added to show in this case we still trim data with time context right.

Reply:
`test_context_adjustment_window_udf_groupby` in `test_timecontext.py`

Reply:
OK